execution/gethexec/sequencer.go

```diff
@@ -52,6 +52,8 @@ var (
 	nonceFailureCacheOverflowCounter        = metrics.NewRegisteredGauge("arb/sequencer/noncefailurecache/overflow", nil)
 	blockCreationTimer                      = metrics.NewRegisteredTimer("arb/sequencer/block/creation", nil)
 	successfulBlocksCounter                 = metrics.NewRegisteredCounter("arb/sequencer/block/successful", nil)
+	nonTimeboostedTxInclusionTimer          = metrics.NewRegisteredTimer("arb/sequencer/tx/nontimeboosted/inclusion", nil)
+	timeboostedTxInclusionTimer             = metrics.NewRegisteredTimer("arb/sequencer/tx/timeboosted/inclusion", nil)
```

Comment:
I think "inclusion" is redundant, how about just `arb/sequencer/tx/[non]timeboosted`?